testing/e2e/skr/skr-binding-test/index.js

```diff
@@ -162,6 +162,32 @@ describe('SKR Binding test', function() {
     }
   });
 
+  it('Should not allow creation of more than 10 SKR bindings', async function() {
+    errorOccurred = false;
+    count = 0;
+    while (!errorOccurred && count < 13) {
```

Comment:
We don't know how many bindings have been created in the previous test before we start this one.

Reply:
Could you add such comment in the code?